TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -31,7 +31,9 @@ const GUI_RECT printinfo_val_rect[6] = {
 };
 
 static uint32_t nextTime = 0;
+static uint32_t nextDrawTime = 0;
 static uint32_t toggle_time = 2000; // 1 seconds is 1000
+static uint32_t drawTime = 500; // 1 seconds is 1000
```

Comment:
use #define for constant values

```suggestion
#define drawTime  500 // 1 seconds is 1000
```

Reply:
Good idea!

Reply:
> use #define for constant values

Done.